docs/software-architecture-and-design.md

```diff
@@ -142,100 +123,341 @@ The `TbotsProto::RobotStatus` protobuf message contains information about the st
 * The capacitor charge on the robot
 * The temperature of the dribbler motor
```

Comment:
idk when this was written but we've never communicated the "temperature of the dribbler motor" 😆